adam/create_gaze_ablation_params.py

```diff
@@ -0,0 +1,77 @@
+from vistautils.parameters import Parameters
+from vistautils.parameters_only_entrypoint import parameters_only_entry_point
+import numpy as np
+
+
+def create_gaze_ablation_entry_point(params: Parameters) -> None:
+    """This function creates all possible gaze ablation param files within a given range"""
+    # get the parameters directory, which must be non-null
+    parameters_dir = params.optional_creatable_directory("parameters_directory")
```

Comment:
nit: prefer https://github.com/isi-vista/vistautils/blob/master/vistautils/parameters.py#L244 and let vistautils handle requiring this param